packages/ndla-ui/src/ProgrammeCard/ProgrammeCard.stories.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import ProgrammeCard from './ProgrammeCard';
+import { defaultParameters } from '../../../../stories/defaults';
+
+export default {
+  title: 'Enkle komponenter/ProgrammeCard',
+  component: ProgrammeCard,
+  tags: ['autodocs'],
+  parameters: {
+    ...defaultParameters,
+  },
+  args: {
+    id: 'test ID',
+    programmeTitle: { title: 'Elektro og datateknologi', language: 'nb-no' },
```

Comment:
Vi bruker 'nb', 'nn' og 'en' i alt vi har av data.
```suggestion
    programmeTitle: { title: 'Elektro og datateknologi', language: 'nb' },
```